DesignSystem/Sources/StudyPreview/MOITStudyPreview.swift

```diff
@@ -0,0 +1,227 @@
+        self.flexRootView.flex.layout()
+        self.clipsToBounds = true
+    }
+    
+    private func configureLayout() {
+        
+        self.flexRootView.flex
+            .direction(.row)
+            .define { (flex) in
+                flex.addItem() // 보이는 뷰
+                    .paddingLeft(16)
+                    .marginRight(10)
+                    .width(100%)
+                    .height(100)
+                    .cornerRadius(30)
+                    .backgroundColor(.white)
+                    .alignItems(.center)
+                    .direction(.row)
+                    .define { flex in
+                        flex.addItem(profileImageView)
+                            .size(60)
+                            .marginRight(10)
+                        
+                        flex.addItem()
+                            .direction(.column)
+                            .marginVertical(14)
+                            .paddingRight(16)
+                            .height(71)
+                            .alignItems(.start)
+                            .define { flex in
+                                flex.addOptionalItem(remainingDateLabel).marginBottom(4)
+                                flex.addItem(studyNameLabel).marginBottom(0)
+                                flex.addItem(studyDescriptionLabel)
+                            }
+                    }
+                
+                
+                flex.addItem(deleteButton) // 옆에 버튼
+                    .backgroundColor(ResourceKitAsset.Color.orange100.color)
+                    .aspectRatio(0.77)
+                    .cornerRadius(20)
+            }
+    }
+    
+    private func configureAttributes(remainingDate: Int, profileURL: URL, studyName: String, studyProgressDescription: String?) {
+        deleteButton.setImage(ResourceKitAsset.Icon.trash.image.withTintColor(.white), for: .normal)
+        deleteButton.addTarget(self, action: #selector(didTapDelete), for: .touchUpInside)
+        flexRootView.button = deleteButton
+        
+        remainingDateLabel = MOITChip(type: .dueDate(date: remainingDate))
+        
+        profileImageView.kf.setImage(
+            with: profileURL,
+            options: [.processor(RoundCornerImageProcessor(cornerRadius: 20))]
+        )
+        studyNameLabel.text = studyName
+        
+        studyDescriptionLabel.text = studyProgressDescription
+        self.flexRootView.flex.markDirty()
+        self.layoutIfNeeded()
+    }
+    
+    @objc private func onPan(_ gestureRecognizer: UIPanGestureRecognizer) {
+        let translation = gestureRecognizer.translation(in: flexRootView)
+        let velocity = gestureRecognizer.velocity(in: flexRootView)
+        
+        switch gestureRecognizer.state {
+        case .began, .changed:
+            if translation.x < 0 {
+                flexRootView.transform = CGAffineTransform(translationX: translation.x, y: 0)
+            }
+        case .ended:
```

Comment:
이 부분 그때 고민이라고 하셨던거 생각나서 좀 고쳐봤는데...
```suggestion
        case .began:
		if translation.x < 0 {
			let translationX = flexRootView.transform.tx < 0 ? flexRootView.transform.tx : translation.x
			
			flexRootView.transform = CGAffineTransform(
				translationX: translationX,
				y: 0
			)
		}
	case .changed:
		if translation.x < 0 && translation.x < flexRootView.transform.tx {
			flexRootView.transform = CGAffineTransform(translationX: translation.x, y: 0)
		}
```
요렇게 고치면

![Simulator Screen Recording - iPhone 14 Pro - 2023-06-12 at 13 40 05](https://github.com/mash-up-kr/MOIT-iOS/assets/22907483/44aabb12-a5f5-4ea6-b759-3d380d0e09d0)

이렇게 되긴하는데 완벽하진 않아서 그냥 참고하시면 좋을 것 같슴다 !